corehq/apps/app_manager/models.py

```diff
@@ -4220,6 +4220,14 @@ def rename_lang(self, old_lang, new_lang):
     def is_remote_app(self):
         return False
 
+    def get_version(self, version_number):
```

Comment:
Can't this use `get_build_doc_by_version`?

Reply:
(And if not, this logic should go in `dbaccessors`.)

Reply:
ah yes. thanks

Reply:
made the change, would just test it again.